kong/clustering/compat/checkers.lua

```diff
@@ -40,37 +40,43 @@ local compatible_checkers = {
 
         if plugin.name == 'ai-proxy' then
           local config = plugin.config
-          if config.model.provider == "gemini" then
+          if config.model.provider == "gemini" or config.model.provider == "bedrock" then
```

Comment:
over time I'd suggest to go for static provider names to avoid typos and easier name-changes.

`local GEMINI_PROVIDER="gemini"`

Reply:
See below, fixed together